saleor/graphql/order/resolvers.py

```diff
@@ -38,3 +58,11 @@ def resolve_shipping_methods(obj, info):
     return qs.applicable_shipping_methods(
         price=obj.get_subtotal().gross.amount, weight=obj.get_total_weight(),
         country_code=obj.shipping_address.country.code)
+
+
+def resolve_order_events(info):
+    # Filter only selected events to be displayed on homepage.
+    types = [
```

Comment:
Do we want types to be hardcoded here? If yes, then maybe we should rename resolver to something more suggestive, for example `resolve_placed_orders`.

Reply:
Those events will be hardcoded for now. Maybe I should rename it to `resolve_homepage_activity` as it is supposed to return some (not all) events suitable for dashboard homepage. Later this would include also product- or user-related events.

Reply:
That is fine. I would just leave a comment explaining why and for what this resolver is, so if some dev would fork project, he would easily find a place where he can customize his own dashboard activity (or remove, if not needed).